usingr.Rmd

```diff
@@ -0,0 +1,359 @@
+
+```{r setup, echo=FALSE, warning=FALSE, purl=FALSE, message=FALSE}
+options(repos = "http://cran.rstudio.com/")
+pkgs <- c("dplyr", "knitr")
+x<-lapply(pkgs, library, character.only = TRUE)
+opts_chunk$set(tidy = FALSE, message = F, warning = F)
+```
+
+# Using R for bioassessment data: What, why, and how
+
+## Lesson Outline
+
+- [Goals and Motivation]
+- [RStudio]
+- [R language fundamentals]
+- [Data structures in R]
+- [Data I/O]
```

Comment:
I would actually say something like "Getting Data In/Out of R". Some folks won't know about I/O.

Reply:
Ah, good call.  I changed this to "Getting your data into R".